jmix-pivottable/pivottable-flowui-kit/src/main/resources/META-INF/frontend/src/pivot-table/jmix-pivot-table.js

```diff
@@ -237,19 +241,26 @@ export class JmixPivotTable extends ElementMixin(DisabledMixin(ThemableMixin(Pol
             "value-a-to-z": "key-a-to-z",
             "value-z-to-a": "value-a-to-z"
         };
-        this.$jQuery(orderElementClassName).val("").html("");
+
+        var pivotTableOutputElement = this._findPivotTableOutputElement();
+
+        pivotTableOutputElement.find(orderElementClassName).val("").html("");
 
         let currentOrder = order.replace(/_/g, '-');
-        this.$jQuery(orderElementClassName).removeClass(nextToCurrent[currentOrder]);
-        this.$jQuery(orderElementClassName).addClass(currentOrder);
+        pivotTableOutputElement.find(orderElementClassName).removeClass(nextToCurrent[currentOrder]);
+        pivotTableOutputElement.find(orderElementClassName).addClass(currentOrder);
     }
 
     _onRendererChange() {
         this._recreatePivot();
     }
 
+    _findPivotTableOutputElement() {
+        return this.$jQuery("div.pivot-table-output-" + this.pivotTableId);
+    }
+
     _recreatePivot() {
-        let outputDiv = this.$jQuery("div.pivot-table-output");
+        let outputDiv = this._findPivotTableOutputElement();
```

Comment:
Obtain `outputDiv` only when it's needed. This method called for the first time before `outputDiv` is created. With your approach `_findPivotTableOutputElement` tries to find and element with `pivot-table-output-undefined` class name